### DISCUSSION TITLE
Code signing

### DISCUSSION THREAD

**Discussion by pandaninjas** [modrinth/.github#96] [Feature requests]
### Is your feature request related to a problem? Please describe.

I cannot include signatures with my binaries in any way, except by including the signature in the release description

### Describe the solution you'd like

2 possible solutions:
1) Allow uploading asc files
2) Have a system that allows you to upload asc files and verifies the binaries

### Describe alternatives you've considered

Current workaround is to include them in the release description.

**Comment by BasiqueEvangelist** [comment]
This is much more important now, considering the current situation with malware in mods.
Would a system that blocks unsigned files/marks them as explicitly unsigned (a la Vigilant Mode) be nice here? Mainly so that it's incredibly noticeable when somebody just doesn't sign a JAR (or uses a bad key)

**Comment by zenfyrdev** [comment]
If something like this is ever implemented, could SSH signatures be also supported? Since git, GitHub and GitLab also support them.
